AzureData/README.md

```diff
@@ -119,12 +119,24 @@ database.get (collectionWithId: id) { r in
 
 #### Delete
 ```swift
-AzureData.delete (collection, from: databaseId) { s in
-    // s == successfully deleted
+AzureData.delete (collection, fromDatabase: databaseId) { r in
```

Comment:
Don't think this override is necessary, if we have the `collection` object, we should be able to just use the generic `delete(resource)` function and don't need the `databaseId`... correct?

Reply:
I agree and thought the same. This override exists on Android. Should we keep it for parity with Android?

Reply:
no, we should ping @NateRickard to remove it from android unless he has a reason not to

Reply:
I think this is the one where the doc was off a bit but let me know if I'm missing something.